instances/latest/commonCoordinateSpaceVersions/P-MarmosetBSC-corT/P-MarmosetBSC-corT_v2012-Interaural-RAS.jsonld

```diff
@@ -0,0 +1,73 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.om-i.org/props/"
+  },
+  "@id": "https://openminds.om-i.org/instances/commonCoordinateSpaceVersion/P-MarmosetBSC_v2012",
+  "@type": "https://openminds.om-i.org/types/CommonCoordinateSpaceVersion",
+  "abbreviation": "P-MarmosetBSC_v2012",
```

Comment:
same abbreviation as CCS: "P-MarmosetBSC" (i.e., version info is not included here)